package EntornosProgramacion.SistemaInventario.dto.request;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;

public record LoginRequest(
    @Email(message = "El email no tiene un formato valido")
    @NotBlank(message = "El email es obligatorio")
    String email,

    @NotBlank(message = "La contrasena es obligatoria")
    String password
) {
}
